honeybee_energy/hvac/heatcool/_base.py

```diff
@@ -63,6 +70,36 @@ def to_ideal_air_equivalent(self):
         i_sys._display_name = self._display_name
         return i_sys
 
+    @property
+    def properties(self):
+        """Get properties for extensions."""
+        return self._properties
+
+    @classmethod
+    def from_dict(cls, data):
+        new_obj = super(_HeatCoolBase, cls).from_dict(data)
+        if 'properties' in data and data['properties'] is not None:
+            new_obj.properties._load_extension_attr_from_dict(data['properties'])
+        return new_obj
+
+    @classmethod
+    def from_dict_abridged(cls, data, schedule_dict):
+        new_obj = super(_HeatCoolBase, cls).from_dict_abridged(data, schedule_dict)
+        if 'properties' in data and data['properties'] is not None:
+            new_obj.properties._load_extension_attr_from_dict(data['properties'])
+        return new_obj
+
+    def to_dict(self, abridged=False):
+        base = super(_HeatCoolBase, self).to_dict(abridged)
```

Comment:
I guess the tests would show this but are you sure that this still returns the correct type of class instance downstream? So this doesn't have to be:
```
base = super(self.__class__, self).to_dict(abridged)
```

Reply:
Ok. Makes sense. And, yes, it has to work in both Python 2 and Python 3, though we are running the tests in both so this hopefully means your particular implementation of super() is OK for this case.

Reply:
I'll edit this, as with the above, to call the Parent directly instead of using super(). That way It'll all be consistent throughout.